pulp_smash/tests/docker/cli/utils.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+"""Common utilities that assist in testing the CLI with docker repositories."""
+from __future__ import unicode_literals
+
+import unittest2
+
+from pulp_smash import cli, config, utils
+
+
+FEED = 'https://index.docker.io'
```

Comment:
As of d93de79f0cec13779a719d45d3a8e4dac1cd34dd, there are now docker feed constants. Can we use one of those constants, or at least make `FEED` private?

By the way, as per our discussion on the mailing lists, I'm not entirely sure if those constants have correct values. It might make more sense to have something like this in the constants file:

```python
DOCKER_V1_FEED_URL = '...'
DOCKER_V2_FEED_URL = DOCKER_V1_FEED_URL
```

Or even something else. Basically: if the docker-related constants don't seem right to you, please speak up and/or submit a pull request.